src/Features/CSharp/Portable/Completion/KeywordRecommenders/ThisKeywordRecommender.cs

```diff
@@ -58,39 +58,26 @@ private bool IsConstructorInitializerContext(CSharpSyntaxContext context)
             return false;
         }
 
-        private static bool IsExtensionMethodParameterContext(CSharpSyntaxContext context, CancellationToken cancellationToken)
+        private static bool IsThisParameterModifierContext(CSharpSyntaxContext context)
         {
             // TODO(cyrusn): lambda/anon methods can have out/ref parameters
```

Comment:
does comment still apply here?

Reply:
➡️ Removed in 1a2e2a6f2c2592d45fc6b337ce5dd212a2c0acb3